### PR TITLE
Towards #3041 remove Task.launchCounter and add list of LocalVolumeIds

### PR DESCRIPTION
**Pull request by meichstedt**

### CODE REVIEW

File: src/main/scala/mesosphere/marathon/core/task/Task.scala [hunk 1]
```diff
@@ -23,7 +22,7 @@ case class Task(
     * Legacy conversion to MarathonTask. Cache result to speed up repeated uses.
     * Should be removed before releasing 0.16.
     */
-  lazy val marathonTask: MarathonTask = TaskSerializer.marathonTask(this)
+  lazy val marathonTask: MarathonTask = TaskSerializer.toProto(this)
```
Comment (kolloch) [inline src/main/scala/mesosphere/marathon/core/task/Task.scala]: nice!

**Comment by kolloch** [comment]
apidoc generation fails (reproduce by `sbt doc`). Otherwise just minor comments, nice work!

File: src/main/proto/marathon.proto [hunk 2]
```diff
@@ -90,6 +90,10 @@ message ResourceRoles {
 }
 
 message MarathonTask {
+  message ReservationWithVolumes {
+    repeated string volumeIds = 1;
```
Comment (meichstedt) [inline src/main/proto/marathon.proto]: full ack; will rename and change to underscore separation instead of camel humps

**Comment by meichstedt** [comment]
adressed your comments – thanks!

**Comment by kolloch** [comment]
LGTM :) :+1:

**Pull request merged**